es66x_revie_fix.patch

```diff
@@ -0,0 +1,106 @@
+Index: es79x/src/main/scala/tech/beshu/ror/es/request/handler/regular/RegularRequestHandler.scala
```

Comment:
looks like a trash ;)

Reply:
yeah, it's a trash :D I used patches to apply my change to every es version and I forgot to remove this file